src/child/selectors/selectors.js

```diff
@@ -27,10 +29,13 @@ export const searchSelector = createCurrentWindowStateSelector(
     }
 );
 
-export const sidebarSelector = createCurrentWindowStateSelector(
-    (state) => {
-        const { sidebar, selection, favourites, windowState } = state;
-        return { sidebar, selection, favourites, windowState };
+export const sidebarSelector = createSelector(
+    [getClosedWindows, getCurrentWindowState],
+    (closedWindows, currentWindowState) => {
+        const closedWindowNames = closedWindows ? Object.keys(closedWindows) : [];
```

Comment:
`closedWindows` is initialised as `{}` and `Object.keys({})` returns `[]` so this isn't needed.